content/en/platform/corda/5.0/deploying-operating/observability/metrics/db-worker.md

```diff
@@ -0,0 +1,77 @@
+---
+date: '2023-06-14'
+version: 'Corda 5.0'
+title: "Database Worker"
+version: 'Corda 5.0'
+menu:
+  corda5:
+    parent: corda5-cluster-metrics
+    identifier: corda5-cluster-db-worker
+    weight: 10000
+section_menu: corda5
+---
+
+# Database Worker
+
+The database worker is the sole worker with access to the database (apart from the crypto worker, which has its
+own dedicated database). As a result, the activities carried out within the database worker pertain to the database.
+
+The database worker is responsible for handling and serving persistence requests originating from various Corda worker types,
+such as the flow worker or the MGM worker. The flow persistence requests metrics presented in this section measure:
+
+* The time taken to handle the flow persistence requests.
+* The time the flow persistence requests remained on Kafka, from the moment they were added by flows until they were received
+  by the database worker (Kafka lag).
+
+The ledger persistence requests metrics measure the time taken to handle the ledger persistence requests.
```

Comment:
Think I commented before that I found this hard to read

Reply:
updated